tests/datasource/fluent/test_pandas_datasource.py

```diff
@@ -476,6 +476,21 @@ def test_dataframe_asset(
     )
 
 
+@pytest.mark.cloud
```

Comment:
We don't mark E2E tests any differently right? Might be helpful to add a comment or update the test name to indicate that this makes external requests and relies on state in our OSS CICD acct in prod